### PR TITLE
Makes augmentations more difficult to maintain

### PR DESCRIPTION
**Pull request by Ikarrus**
Medical has had several overhauls, but none of them touched augmentations, so they became almost a no-contest better alternative than regular limbs in virtually every aspect.

Repairing augmentations is no longer instant, and will take more steps than healing with medical items on biological parts.

### CODE REVIEW

File: code/modules/power/cable.dm [hunk 1]
```diff
@@ -514,7 +514,9 @@ var/global/list/datum/stack_recipe/cable_coil_recipes = list ( \
 
 	var/obj/item/organ/limb/affecting = H.get_organ(check_zone(user.zone_sel.selecting))
 	if(affecting.status == ORGAN_ROBOTIC)
-		item_heal_robotic(H, user, 0, 30)
+		user.visible_message("[user] starts to fix some of the wires in [H]'s [affecting.getDisplayName()].", "<span class='notice'>You start fixing some of the wires in [H]'s [affecting.getDisplayName()].</span>")
+		if(!do_mob(user, H))	return
```
Comment (Iamgoofball) [inline code/modules/power/cable.dm]: don't you need to set a time for this?

Comment (Ikarrus) [inline code/modules/power/cable.dm]: Default is 3 seconds. Which I think works.

**Comment by Boredone** [comment]
As a note, I believe the EMP damage procs for every augged limb you have when emped. So a full EMP could out-right crit a guy fully augged.
Not complaining, just making you aware if you weren't already.

**Comment by Ikarrus** [comment]
Fully aware.

A fully aug'd guy already has a pretty huge advantage with the built-in space suit. I figure it'd only be fitting it would come with an equally powerful downside, as rare as it is.

**Comment by freerealestate** [comment]
Augmentations are a pain in the ass to perform and require a willing and able surgeon. They aren't used very often. What prompted this PR?

**Comment by Ikarrus** [comment]
Not everything we do has to have an ulterior motive to it @freerealestate.  My reasons are in the description.

**Comment by Iamgoofball** [comment]
@freerealestate 

maybe you should take the tinfoil hat off sometimes man

it's good for you

**Comment by Ikarrus** [comment]
Not everything we do has to have some kind of ulterior motive or grudge behind it  @freerealestate

**Comment by freerealestate** [comment]
Augmentations rarely pose a problem because they never get used due to
being so time-consuming to give. The reason given in the OP isn't very
detailed or logical, so I'm asking what brought them to your attention.

I never use augmentations, I just don't see the purpose in nerfing them.

**Comment by SconesC** [comment]
There is now no reason to take augs, sorry

**Comment by Ikarrus** [comment]
Discussions with people who do use them who say they literally have no downsides, which is true

- Virtually infinite healing with common items found pretty much everywhere
- no delay on use meaning you can instantly heal yourself mid combat
- built in damage resistance
- built in space suit
- their one supposed weakness (EMP) is a joke

They're still pretty good. You may be overestimating what my PR changes.

**Comment by SconesC** [comment]
The wires/welding change is fine

It's the 200% buff to EMP damage I'm worried about here

Sadly this change just pushes them farther into the corner of extreme/rarely used they were already in

**Comment by Ikarrus** [comment]
What about it worries you?  If you only have a couple of augmentations, say, to your head and chest, an EMP blast would still be noticable, but it wouldn't kill you. 

This is also assuming your attacker knows that you're augmented (which is easily concealable) and would use EMP attacks, which are quite rare themselves.

**Comment by SconesC** [comment]
Hilariously enough, what I've found (with excessive use of augs) is that your greatest foe is friendly fire. 

It just seems like a "ain't broke don't fix it" situation.

Less people will use augs, and they were already underused. It's pushing a backburner feature practically off the stove.

**Comment by Ikarrus** [comment]
I'm willing to discuss with you the EMP damage and I'm more than willing to reduce the damage if there was an actual concern. 

Just because a feature is relatively unknown isn't a good enough reason imo. It'd just remain within the realms of secret monkey techniques that only players like you would be savvy enough to exploit. If anything, it just means we may need to think about how to make it less obtuse to get augmented. 

I personally don't think EMP is used enough to really have such a dramatic response over, but I mean if you were to say that getting EMP'd shouldn't outright crit you regardless of the situation I'd be fine with that.

**Comment by SconesC** [comment]
Conflict of design opinions in that case, I guess

**Comment by Lumipharon** [comment]
emp's already assblast augged people because it stuns.
When augs first came in though, it was 20 brute PER AUGMENT, so a fully augged guy got insta critted by an emp.

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 2]
```diff
@@ -228,10 +228,10 @@ emp_act
 				informed = 1
 			switch(severity)
 				if(1)
-					L.take_damage(0,10)
+					L.take_damage(0,30)
```
Comment (Dorsisdwarf) [inline code/modules/mob/living/carbon/human/human_defense.dm]: so the full 6 augments will deal a massive 180 damage to you whenever some git fires the ion rifle at anything within 2 tiles of you?

Augs aren't /that/ good

**Comment by GunHog** [comment]
Do not forget that augs cannot be healed by medical items, which means that they will confuse anyone trying to help unless they know they are dealing with an aug. If we had more augmentations that included tools/weapons or such, I could see this a bit more justified. The few times I have gotten to augment someone, I have *always* done the full set.

60 brute for heavy EMP certainly is more than "basically nothing", 180 damage is far too much. I would personally find this nerf more acceptable if you removed the EMP damage change, and went with only the healing elements, as that is simply bringing them in line with healing methods for normal humans.

**Pull request merged**